docs/README.md

```diff
@@ -250,9 +250,10 @@ import { Tab, Tabs } from '~/components/plugins/Tabs';
     /* @end */
     };
 
-    </Tab>
+</Tab>
 </Tabs>
 ```
+n.b. The components should not be indented or they will not be parsed correctly.
```

Comment:
n.b. ?

Reply:
https://en.wikipedia.org/wiki/Nota_bene
More or less FYI, but in latin :p